dotnet/src/Connectors/Connectors.OpenAI/OpenAIServiceCollectionExtensions.cs

```diff
@@ -232,17 +242,28 @@ public static class OpenAIServiceCollectionExtensions
         string apiKey,
         string? orgId = null,
         string? serviceId = null,
+        HttpClient? httpClient = null) => AddOpenAIChatCompletion(builder, new() { ModelId = modelId, ApiKey = apiKey, Organization = orgId }, serviceId, httpClient);
```

Comment:
Looks like similar change in a couple of places, just wanted to double check: if original method is `AddOpenAITextGeneration`, should we re-use the same `AddOpenAITextGeneration` overload method instead of `AddOpenAIChatCompletion`?
```suggestion
        HttpClient? httpClient = null) => AddOpenAITextGeneration(builder, new() { ModelId = modelId, ApiKey = apiKey, Organization = orgId }, serviceId, httpClient);
```